src/HTTPSConnection.cpp

```diff
@@ -109,7 +109,11 @@ size_t HTTPSConnection::writeBuffer(byte* buffer, size_t length) {
 }
 
 size_t HTTPSConnection::readBytesToBuffer(byte* buffer, size_t length) {
-  return SSL_read(_ssl, buffer, length);
+  int ret = SSL_read(_ssl, buffer, length);
+  if (ret < 0) {
+	HTTPS_LOGD("SSL_read error: %d",  SSL_get_error(_ssl, ret));
```

Comment:
just being a bit picky about indentation:

```suggestion
    HTTPS_LOGD("SSL_read error: %d",  SSL_get_error(_ssl, ret));
```